### PR TITLE
Feature/cso specs

### PR DESCRIPTION
**Pull request by bvizureanu**
### What does it fix?

Related to #29

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -19,9 +19,6 @@
 	// Add the IDs of extensions you want installed when the container is created.
 	"extensions": [
 		"ms-dotnettools.csharp",
-        "ms-vscode-remote.remote-containers",
-        "ms-vscode-remote.remote-wsl",
-        "ms-azuretools.vscode-docker",
```
Comment (andreifloroiu) [inline .devcontainer/devcontainer.json:24]: Why remove these?

File: specs/resources/cso-id.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# get {id}
+get:
+  operationId: getCso
+  summary: Gets a CSO based on id
+  description: Gets a CSO based on Id
+  tags:
+    - cso
+    - get
+  parameters:
+    - $ref: '../alex-api.yml#/components/parameters/IdParameter'
```
Comment (andreifloroiu) [inline specs/resources/cso-id.yml:10]: Why ID parameter? We get and edit same CSO organization.

File: specs/parameters/id-parameter.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+in: path
+name: "id"
+schema:
+  type: string
+required: true
+description: Identifier for entities
```
Comment (andreifloroiu) [inline specs/parameters/id-parameter.yml:6]: Duplicate - there is already an existing common-id-parameter.yml. We can stick with that or we can refactor that for this name.

File: specs/schemas/cso.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  registryNumber:
+    type: string
+  legalRepresentativeName:
+    type: string
+  legalRepresentativePhone:
+    type: telephone
+  legalRepresentativeEmail:
+    type: email
+  legalRepresentativeAddress:
+    type: string
```
Comment (andreifloroiu) [inline specs/schemas/cso.yml:19]: We can create a nested object, like in sponsor schema.

File: specs/alex-api.yml [hunk 5]
```diff
@@ -15,6 +15,10 @@ paths:
     $ref: './resources/accounts.yml'
   /v1/accounts/{id}:
     $ref: './resources/account.yml'
+  /v1/cso:
+    $ref: './resources/cso.yml'
+  /v1/cso/{id}:
```
Comment (andreifloroiu) [inline specs/alex-api.yml:20]: Why this route, with ID? Related to the other comment.

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (bvizureanu) [reply .devcontainer/devcontainer.json:24]: inside the container they have no use (these are specs for the container environment)

File: specs/parameters/id-parameter.yml (hunk 3 above)
Comment (bvizureanu) [reply specs/parameters/id-parameter.yml:6]: This was created in order to be generic (as it has the same type, scope and everything)

**Pull request closed without merging**